src/widgets/hierarchical-menu/hierarchical-menu.tsx

```diff
@@ -358,6 +359,7 @@ const hierarchicalMenu: HierarchicalMenuWidget = function hierarchicalMenu(
       showMore,
       showMoreLimit,
       sortBy,
+      facetOrdering,
       transformItems,
     }),
     $$widgetType: 'ais.hierarchicalMenu',
```

Comment:
I noticed something that looks like a quirk, it's probably not related to this change, but i'm hoping you can explain it.


![2021-06-18 18 26 38](https://user-images.githubusercontent.com/2982512/122591143-d5f66800-d062-11eb-8fcd-8561eb7c4a61.gif)

I think it maybe because it fallbacks to count sorting, but the whole behaviour feels quirky, doesn't it?

![Screenshot at Jun 18 18-25-48](https://user-images.githubusercontent.com/2982512/122591182-e3abed80-d062-11eb-849f-4abaccd7de99.png)


```js
instantsearch.widgets.hierarchicalMenu({
     container,
      attributes: ['hierarchicalCategories.lvl0', 'hierarchicalCategories.lvl1', 'hierarchicalCategories.lvl2'],
}),
       
```

Reply:
This happens because the count swaps to 1 at the end (not sure what causes that, but likely some engine bug), for a hierarchical facet `alpha` makes much more sense as the sortRemainingBy. We'll make sure to make that clear in the guides!